lib/pages/home/widgets/home_title.dart

```diff
@@ -50,6 +50,38 @@ class HomeTitle extends StatelessWidget {
                   decoration: TextDecoration.underline,
                 ),
               ),
+              Text(
+                Strings.period,
+                style: Styles.textStyleLargeParagraph.copyWith(
+                  color: CustomColors.light75,
+                ),
+              ),
+            ],
+          ),
+          const SizedBox(height: 24),
+          Wrap(
+            alignment: WrapAlignment.center,
+            children: [
+              Text(
+                Strings.sourceCode,
+                style: Styles.textStyleLargeParagraph.copyWith(
+                  color: CustomColors.light75,
+                ),
+              ),
+              UrlText(
+                url: Url.githubLink,
+                text: Strings.here,
+                style: Styles.textStyleLargeParagraph.copyWith(
+                  color: CustomColors.light75,
+                  decoration: TextDecoration.underline,
+                ),
+              ),
+              Text(
+                Strings.period,
```

Comment:
This widget is repetitive already 3 times (at least in this PR.) Better to have 1 reused.